### DISCUSSION TITLE
XDM v8.0.11 Beta Won't Shut Down

### DISCUSSION THREAD

**Discussion by ColtonYYZ** [subhra74/xdm#968] [Q&A]
Hello!  I have the 8.0.11 Beta version installed (I know beta isn't final release) but I am hoping someone knows a tweak or trick to keep it shut down once I have manually selected Exit in the main window under settings? 

My browser ext is turned off, yet it still prompts me to download videos it's found, so this is another bug but not my complaint. 

Once I exit the program (to shut it down/turn it off) it auto starts up again. And yes, I have that option UNCHECKED for it to auto start with Windows.

Any ideas what I can do, or is it just that; a bug, and I need to wait until a fix is implemented?

**Comment by subhra74** [comment]
Please try latest version 8.0.18, and let me know if this solves the issue. Please note after installing the new version you need to go to browser integration page and click on appropriate browser integration button to fix the browser integration issues. Also please deselect the checkbox 'Show download video notifications' in browser monitoring page in XDM. Captured videos will be visible in browser extension itself.
https://github.com/subhra74/xdm-experimental-binaries/releases/tag/8.0.18-beta
